src/box/lua/upgrade.lua

```diff
@@ -1517,6 +1530,405 @@ local function upgrade()
     upgrade_from(get_version())
 end
 
+--------------------------------------------------------------------------------
+-- Downgrade part
+--------------------------------------------------------------------------------
+
+--------------------------------------------------------------------------------
+-- Tarantool 2.9.1
+--------------------------------------------------------------------------------
+
+local function restore_sql_builtin_functions(issue_handler)
```

Comment:
General remark: would be nice to leave a reference to the corresponding commit for each downgrade function.

Reply:
Done.